gradle/play-publisher.gradle

```diff
@@ -0,0 +1,7 @@
+apply plugin: 'com.github.triplet.play'
+
+play {
+    track = findProperty("releaseTrack") as String ?: "internal"
+    serviceAccountCredentials = file("test-app-play-publisher.json")
```

Comment:
Where is this file supposed to come from?

Reply:
its in ci environment of internal branch.